operator-framework-junit5/src/main/java/io/javaoperatorsdk/operator/junit/OperatorExtension.java

```diff
@@ -118,6 +139,10 @@ public Reconciler getFirstReconciler() {
         .findFirst().orElseThrow();
   }
 
+  public boolean isLocal() {
+    return this.localOperatorDeployment;
```

Comment:
wonder if this can be implicit, if you have a operator deployment, then I guess it is local